### PR TITLE
NFS Driver FileshareAcl AccessTo is limited to one user

### PR DESCRIPTION
**Pull request by Shruthi-1MN**
**What this PR does / why we need it**:
NFS Driver FileshareAcl AccessTo is limited to one user

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
```

### CODE REVIEW

File: contrib/drivers/filesharedrivers/nfs/nfs.go [hunk 1]
```diff
@@ -86,40 +85,39 @@ func (*Driver) Unset() error { return nil }
...
 
 	for _, value := range accessCapability {
-		if value == "w" {
-			access = AccessLevelRw
-			break
-		} else {
-			access = AccessLevelRo
+		value = strings.ToLower(value)
+		if (utils.Contains(permissions, value)){
+				access += value[:1]
```
Comment (leonwanghui) [inline contrib/drivers/filesharedrivers/nfs/nfs.go]: The code style is not correct, please run `make goimports` to format it.

File: contrib/drivers/filesharedrivers/nfs/nfs.go [hunk 2]
```diff
@@ -86,40 +85,39 @@ func (*Driver) Unset() error { return nil }
...
+		}else{
+			log.Error("invalid permission : ", value)
```
Comment (leonwanghui) [inline contrib/drivers/filesharedrivers/nfs/nfs.go]: Please remove the blank spaces here.

File: contrib/drivers/filesharedrivers/nfs/nfs.go [hunk 3]
```diff
@@ -43,12 +44,10 @@ const (
 const (
 	KLvPath            = "lvPath"
 	KLvsPath           = "lvsPath"
-	KFileshareName     = "nfsFileshareName"
+	KFileshareName     = "Name"
```
Comment (leonwanghui) [inline contrib/drivers/filesharedrivers/nfs/nfs.go]: s/Name/name

File: contrib/drivers/filesharedrivers/nfs/nfs.go (hunk 1 above)
Comment (Shruthi-1MN) [reply contrib/drivers/filesharedrivers/nfs/nfs.go]: Done

File: contrib/drivers/filesharedrivers/nfs/nfs.go (hunk 2 above)
Comment (Shruthi-1MN) [reply contrib/drivers/filesharedrivers/nfs/nfs.go]: Done

File: contrib/drivers/filesharedrivers/nfs/nfs.go (hunk 3 above)
Comment (Shruthi-1MN) [reply contrib/drivers/filesharedrivers/nfs/nfs.go]: Done

File: pkg/model/proto/model.proto [hunk 4]
```diff
@@ -728,7 +728,7 @@ message DetachVolumeOpts {
     string context = 4;
 }
 
-// CreateFileShareAclOpts is a structure which indicates all required properties for creating a file share.
+// DreateFileShareAclOpts is a structure which indicates all required properties for creating a file share.
```
Comment (leonwanghui) [inline pkg/model/proto/model.proto]: s/DreateFileShareAclOpts/DeleteFileShareAclOpts

Comment (Shruthi-1MN) [reply pkg/model/proto/model.proto]: Done

**Comment by Shruthi-1MN** [comment]
> Please remove all these binary files

Removed

**Pull request merged**